Extensions/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PageLocalization/Module.jsx

```diff
@@ -47,7 +48,7 @@ class Module extends Component {
                 </div>}
                 {!module.IsDeleted && !isDefault && <div className="icons-container">
                     <span
-                        className={`icon float-left ${(module.IsLocalized ? " blue" : "")}`}
+                        className={`icon float-left ${(module.IsShared ? " disabled" : (module.IsLocalized ? " blue" : ""))}`}

```

Comment:
If a module is both shared and localized, we _don't_ want the `blue` class, only `disabled`, is that right?

Reply:
yes, correct. Blue class just highlights the current state of a localization button/link. By default this link is a blue when module is not localized, and brightly blue when it is localized (blue class makes it brightly blue).
But in case of shared module we just want to disable (grayed out) it completely. So if we still want to provide information whether it is localized or not it is hard to do that. If button will not be grayed out but we just unbind all events, so users will confuse why it is clickable but nothing is happening. 
Exactly for this purposes we added a `Shared` label, so ideally users should go to shared module settings and proceed with required actions on a module level.

Reply:
The point is it is hard to handle `disabled + blue` and `disabled + brightly blue`. 
So the decision was to skip blue indicator and disable it completely.